### PR TITLE
Refactor postprocessor's train entrypoint

### PR DESCRIPTION
**Pull request by AlekseySh**

### CODE REVIEW

File: tests/test_examples/test_runs_via_configs/configs/train_postprocessor.yaml [hunk 1]
```diff
@@ -13,6 +13,10 @@ num_workers: 0
 max_epochs: 2
 valid_period: 1
 
+# EXTRACTION
+
+embeddings_cache_dir: /tmp
```
Comment (DaloroAT) [inline tests/test_examples/test_runs_via_configs/configs/train_postprocessor.yaml:18]: Do we need not a `null` value of `embeddings_cache_dir` for tests?

File: examples/sop/configs_experimental/train_postprocessor_sop.yaml [hunk 2]
```diff
@@ -55,8 +72,31 @@ sampler:
...
-postprocessing_top_n: 5
+transforms_train:
+  name: augs_hypvit_torch
+  args:
+    im_size: 224
+    min_scale: 0.8
+
+# VALIDATION
+
+batch_size_infernce: 128
```
[the remark this answers is not in the document]
Comment (AlekseySh) [reply examples/sop/configs_experimental/train_postprocessor_sop.yaml:83]: thx

File: tests/test_examples/test_runs_via_configs/configs/train_postprocessor.yaml (hunk 1 above)
Comment (AlekseySh) [reply tests/test_examples/test_runs_via_configs/configs/train_postprocessor.yaml:18]: this thing here is to check that saving is working (not to for optimization)

File: examples/inshop/configs_experimental/train_postprocessor_inshop.yaml [hunk 3]
```diff
@@ -55,8 +67,31 @@ sampler:
...
+# VALIDATION
+
+batch_size_infernce: 128
+
+postprocessor:
+  name: pairwise_images
+  args:
+    top_n: 5
+    pairwise_model: ${pairwise_model}
+    transforms:
```
[the remark this answers is not in the document]
Comment (AlekseySh) [reply examples/inshop/configs_experimental/train_postprocessor_inshop.yaml:85]: they may be different in general

**Pull request merged**